hnn_core/network_models.py

```diff
@@ -27,14 +29,15 @@ def default_network(params=None, add_drives_from_params=False):
     -------
     net : Instance of Network object
         Network object used to store the default network.
-        All connections defining the default network will be
-        appeneded to net.connectivity.
 
     Notes
     -----
-    The default network recreates the model from Jones et al. 2009.
+    The default network builds upon the model from Jones et al. 2009 by adding
+    a distance dependent distribution of calcium channels along L5 pyramidal
+    cell apical dendrites. This addition better reflects the current
+    understanding pyramidal cell electrophysiology.
```

Comment:
Maybe you can have a "See Also" link that refers to the Jones model? Like here

https://github.com/mne-tools/mne-python/blob/eaeef19153154c2e087961fd6d1fdcc6613d0b88/mne/preprocessing/ssp.py#L238-L241

when you build it in sphinx, it should link to the API page of that model